packages/ppx_rapper/ppx_rapper.3.1.0/opam

```diff
@@ -33,7 +33,7 @@ dev-repo: "git+https://github.com/roddyyaga/ppx_rapper.git"
 url {
   src: "https://github.com/roddyyaga/ppx_rapper/archive/3.1.0.tar.gz"
   checksum: [
-    "md5=d6ce377b4a0d8c1c8f9fefd643c35282"
```

Comment:
Thanks. We should check if the old tarball is in the cache and either use that or check if anything has changed before this is merged. I will have a look at it asap

Reply:
Thank you so much 🙏